linkerd/service-profiles/src/client.rs

```diff
@@ -256,7 +256,7 @@ where
         mut self: Pin<&mut Self>,
         cx: &mut Context<'_>,
     ) -> Poll<Result<Profile, Error>> {
-        let span = info_span!("poll_profile");
+        let span = debug_span!("poll_profile");
```

Comment:
this one honestly seems like it should probably have been at trace from day one, that seems pretty verbose...